gooddata-java-model/src/main/java/com/gooddata/sdk/model/settings/SettingItem.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2007-2021, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.sdk.model.settings;
+
+import com.fasterxml.jackson.annotation.*;
+import com.gooddata.sdk.common.util.GoodDataToStringBuilder;
+
+import static com.gooddata.sdk.common.util.Validate.notEmpty;
+import static com.gooddata.sdk.common.util.Validate.notNullState;
+
+@JsonTypeName("settingItem")
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SettingItem {
+
+    private final String name;
+    private final String source;
+    private String enabled;
+
+    @JsonIgnore
+    private Links links;
+
+    /**
+     * Creates new setting item with given value and default source (catalog).
```

Comment:
Really with default source? It seems that user can select whatever source he wants ;)

Reply:
Fixed